ontology/yaml/resources/HVAC/entity_types/PMP.yaml

```diff
@@ -43,6 +43,21 @@ PMP_SS_VSC_WDPM:
   - VSC
   - WDPM
 
+PMP_WDT:
+  id: "1112887574847586304"
+  description: "Variable speed pump with differential temperature across water"
+  is_canonical: true
+  opt_uses:
+  - control_mode
+  - schedule_run_command
+  - run_command
+  - run_status
+  - run_mode
+  - lost_power_alarm
+  implements:
+  - PMP
+  - WDT
```

Comment:
Differential temperature across the pump? Is it measuring the temperature increase in the water as it goes through the pump?

Reply:
Description updated.

Reply:
are the temps across the pump? if not they should be mapped to the system.